### DISCUSSION TITLE
AIR 50.2 - Drag and Drop - Windows 11

### DISCUSSION THREAD

**Discussion by zabdoul60460** [airsdk/Adobe-Runtime-Support#2990] [General]
Hi,
Do you know why since I upgrade Windows and AIR SDK and runtime, drag and drop doesn't works.
Indeed a red [Ø] appear on target.
The same code works very well with Mac OS and it worked before the update.
Many thanks in advance.
Best regards.

**Comment by ylazy** [comment]
Drag & drop will not work if your app is running in elevated mode. In other words, if you "run as administrator" or config your app to run as administrator, drag - drop will not work and the red [Ø] will appear.

I think the issue is not related to Windows or Air Runtime, since my apps are working fine with the latest air runtime and windows version.

**Reply by pol2095** [reply]
I tried, drag in gesture works on Windows

![dragin](https://github.com/airsdk/Adobe-Runtime-Support/assets/6962232/6b4722c0-14c3-4d05-855c-86d8394cc9ca)
